portal/src/main/webapp/js/src/oncoprint/webgl/src/js/oncoprintmodel.js

```diff
@@ -916,7 +928,7 @@ var OncoprintModel = (function () {
     }
     
     OncoprintModel.prototype.getContainingTrackGroup = function (track_id) {
```

Comment:
Note: I subtly changed the behaviour of this method without changing the name. It now no longer returns all track IDs between two visible ‘paragraph breaks’, but only those at the same ‘depth’ of expansion, those that a track should consider when moving. This only affects tracks in groups that do have expansion tracks, so it shouldn't affect existing code outside of the gene set branch. The `moveTrack` method has changed accordingly to expect a target track ID on the same level of expansion, and will transparently place your track (and its expansions) under the bottommost expansion of that sibling.